_posts/2020-03-30-Steering-Group-executive-summary,-action-items,-and-minutes.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Steering Group executive summary, action items, and minutes
+author: Franklin Feingold
+display: true
+---
+
+# Steering Group executive summary, action items, and minutes
+
+Date: Monday March 23, 2020
+
```

Comment:
no more "attended" lines?

Reply:
Franklin and I discussed (shortly following the online meeting, for which I was too late due to the change in daylight savings time) that it is not needed to add this to the online minutes. We do plan to keep it in the detailed (and more messy) agenda+notes+minutes on google doc. If you or people from the @steering team feel different about this, please let us know.

Reply:
Sorry André, in case you happen to look here. I incorrectly used the "steering" mention, which caused you to be included instead of our BIDS steering group team. I have now edited the message.

Reply:
okay, I was just asking for consistency reasons :-) OK from my side to have this in the more detailed/messy notes only